verification/curator-service/ui/src/components/LinelistCaseRow.tsx

```diff
@@ -11,14 +14,18 @@ export interface Case {
 }
 
 export default class LinelistCaseRow extends React.Component<RowProps, {}> {
+
     render() {
         const c = this.props.case;
+        const bg = this.props.background;
         return (
-            <tr>
-                <td>{c._id}</td>
-                <td>{new Date(c.date).toDateString()}</td>
-                <td>{c.outcome}</td>
-            </tr>
+            <TableRow key={c._id} style={{ background: bg }}>
+                <TableCell component="th" scope="row">
+                    {c._id}
+                </TableCell>
+                <TableCell>{new Date(c.date).toDateString()}</TableCell>
```

Comment:
use the new helper you created? guess you'll h ave to merge anyway :) sorry!

Reply:
Yeah, I'll merge tomorrow after the demo (since there's a 90% chance the merge'll get goofed).